lib/cisco_node_utils/cmd_ref/evpn_multisite.yaml

```diff
@@ -0,0 +1,17 @@
+# evpn_multisite.yaml
+---
+_exclude: [ios_xr, N3k, N5k, N6k, N7k, N9k-F]
+
+_template:
+  get_command: "show running-config"
```

Comment:
Is there a simple command like "show running-config evpn" or "show running-config | section evpn" or something like that. That way it reduces the amount of output this produces.

Reply:
Modified this to 'show running-config | section multisite'